packages/api-v2/src/student/student.service.ts

```diff
@@ -71,7 +71,7 @@ export class StudentService {
 
   async update(
     uuid: string,
-    updateStudentDto: UpdateStudentDto
+    updateStudentDto: Partial<Student>
```

Comment:
if all the fields in `Student` are optional now, then what's the difference between this and `Partial<Student>`?